src/UI/Components/CardsCarousel/DebitCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { View, Text, TouchableOpacity, Image, ImageBackground } from 'react-native';
+import styles from './styles';
```

Comment:
why are we keeping all styles at same place? move it to a separate style for dashboard